tools/wave/requirements.txt

```diff
@@ -1,2 +1,2 @@
-ua-parser==0.18.0
-python-dateutil==2.9.0.post0
+ua-parser==0.8.0
+python-dateutil==2.8.1
```

Comment:
was this change intentional? the previous versions [were set by dependabot](https://github.com/web-platform-tests/wpt/commits/master/tools/wave/requirements.txt), so I assume this will just cause dependabot to issue a follow-up PR